changelog_unreleased/javascript/12779.md

```diff
@@ -0,0 +1,75 @@
+#### Make more callee patterns recognized as "test call callee". (#12779 by @HosokawaR)
+
```

Comment:
Can you add description for real-world use case? e.g.

```suggestion
Supports test calls like [Playwright `test.describe`](https://playwright.dev/docs/api/class-test#test-describe).
```

Reply:
Certainly, It's more kind !
I followed your suggestion.